internal/bootfiles/bootfiles.go

```diff
@@ -55,3 +60,30 @@ func (b *BootFiles) DownloadBootFiles(ctx context.Context, params operations.Dow
 	return filemiddleware.NewResponder(operations.NewDownloadBootFilesOK().WithPayload(reader),
 		objectName, contentLength)
 }
+
+func (b *BootFiles) UploadBootFiles(ctx context.Context, params operations.UploadBootFilesParams) middleware.Responder {
+	log := logutil.FromContext(ctx, b.log)
+
+	// Create/Update version in DB
+	version := *params.UploadBootFilesParams.OpenshiftVersion
+	tx := b.db.Begin()
+	if err := tx.Save(&version).Error; err != nil {
```

Comment:
Updating an existing supported version could be risky. I think we should think about all the scenarios and if we want to allow it.
i.e.
1. Cluster register. Generates an ISO. Download to 3 hosts.
2. RHCOS has been updated.
3. New hosts are added to the same cluster with the new RHCOS.

Reply:
Changed impl to support OPENSHIFT_VERSIONS, so use only when overriding the defaults.